File: SimpleExoPlayer/app/src/main/java/com/wildanka/simpleexoplayer/MainActivity.java
package com.wildanka.simpleexoplayer;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Button btnExo = findViewById(R.id.btn_go_to_exo);

        btnExo.setOnClickListener(v -> {
            startActivity(new Intent(MainActivity.this, ExampleMediaPlayerActivity.class));
        });
    }
}

File: VideoPlayer/app/src/main/java/com/wildanka/videoplayerapp/MainActivity.kt
package com.wildanka.videoplayerapp

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btn_show_video.setOnClickListener(View.OnClickListener {
            val intent = Intent(this, VideoDetailActivity::class.java)
            startActivity(intent)
        })
    }
}
